lib/capybara/selenium/driver.rb

```diff
@@ -460,7 +460,7 @@ def find_context
   end
 
   def active_element
-    evaluate_script("document.activeElement")
+    @browser.switch_to.active_element
```

Comment:
@twalpole I've swapped this with the Selenium method, though I'm not sure of the cross-platform implications of `@browser.switch_to`.

Reply:
hmmm -- I'll have to go look at the selenium code to see exactly what this is doing

Reply:
Found some time, looks fine